tensorboard/plugins/debugger/tf_debugger_dashboard/tf-tensor-value-view.html

```diff
@@ -0,0 +1,348 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-toast/paper-toast.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-imports/lodash.html">
+<link rel="import" href="tf-debugger-line-chart.html">
+
+<!--
+  Offers a UI for displaying the value of a single tensor.
+-->
+<dom-module id="tf-tensor-value-view">
+  <template>
+    <paper-toast id="tensorValueToast" text="" always-on-top></paper-toast>
+    <table class="tensor-value-view-table">
+      <tr>
+        <td colspan="2">
+          <div>
+            <paper-input
+              class="inline"
+              id="tensor-name"
+              value="[[tensorName]]"
+              readOnly="true">
+            </paper-input>
+            <span
+              class="value-view-close-button"
+              id="value-view-close-button"
+              on-click="closeButtonCallback">
```

Comment:
Isn't on-tap how Polymer does this?

Please note a clickable `<span>` is not accessible.

Please use a `<button>` e.g.

```
    <button aria-label="Toggle actual size"
            id="main-image-container"
            on-tap="_handleTap"></button>
```

Or better yet, consider a `<paper-icon-button>` since it almost certainly has one with the `x` icon.